recipes/webhelpers2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "webhelpers2" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikeorr/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ebb7ce6239cb57ddcdb532c72172e28fac93315c7ea234439a821f4c443ba54e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - markupsafe >=0.9.2
+    - six >=1.4.0
+
+test:
+  requires:
+    - pytest
+  imports:
+    - {{ name }}
+  commands:
+    - pytest --pyargs {{ name }}
+
+about:
+  home: https://webhelpers2.readthedocs.io/en/latest
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Convenience functions for generating html'
+  description: |
+    WebHelpers2 is the successor to the widely-used WebHelpers utilities.
+    It contains convenience functions to make HTML tags, process text,
+    format numbers, do basic statistics, work with collections, and more.
+  doc_url: https://webhelpers2.readthedocs.io/en/latest
+  dev_url: https://github.com/mikeorr/WebHelpers2
+
+extra:
+  recipe-maintainers:
+    - ChrisBarker-NOAA
+    - mikeorr
```

Comment:
@mikeorr Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes, I will maintain it.